Status Flag 'Pull Request AutoTester' - Jenkins Testing: 1 or more Jobs FAILED

Note: Testing will normally be attempted again in approx. 4 Hrs. If a change to the PR source branch occurs, the testing will be attempted again on next available autotester run.

<details>
<summary>
  <b>Pull Request Auto Testing has FAILED</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Blake</h3>
<ul>
  <li>Build Num: 2718</li>
  <li>Status: FAILED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>Non-B4B;shoc;AT: RETEST;AT: AUTOMERGE</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1392</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_BRANCH</td>
    <td>bogensch/thlm_def</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>c1d48f648351c6094255047fdaa13420e42c6c07</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>04d8402516797810a851a1cb90b02930ed9b4ae6</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Weaver</h3>
<ul>
  <li>Build Num: 2623</li>
  <li>Status: FAILED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>Non-B4B;shoc;AT: RETEST;AT: AUTOMERGE</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1392</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_BRANCH</td>
    <td>bogensch/thlm_def</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>c1d48f648351c6094255047fdaa13420e42c6c07</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>04d8402516797810a851a1cb90b02930ed9b4ae6</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: SCREAM_PullRequest_Autotester_Mappy</h3>
<ul>
  <li>Build Num: 2039</li>
  <li>Status: FAILED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>PR_LABELS</td>
    <td>Non-B4B;shoc;AT: RETEST;AT: AUTOMERGE</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>1392</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_BRANCH</td>
    <td>bogensch/thlm_def</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_SOURCE_SHA</td>
    <td>c1d48f648351c6094255047fdaa13420e42c6c07</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_BRANCH</td>
    <td>master</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_REPO</td>
    <td>https://github.com/E3SM-Project/scream</td>
  </tr>

  <tr>
    <td>SCREAM_TARGET_SHA</td>
    <td>04d8402516797810a851a1cb90b02930ed9b4ae6</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>SCREAM</td>
  </tr>
</table>
</details>


<details>
<summary>
  <b>Console Output (last 100 lines) : SCREAM_PullRequest_Autotester_Blake # 2718</b> (click to expand)
</summary>
<pre>
<code>
182/188 Test #186: homme_shoc_cld_spa_p3_rrtmgp_np1_vs_np2_bfb ...   Passed    0.17 sec
        Start 187: homme_shoc_cld_spa_p3_rrtmgp_np1_vs_np3_bfb
183/188 Test #187: homme_shoc_cld_spa_p3_rrtmgp_np1_vs_np3_bfb ...   Passed    0.15 sec
        Start 188: homme_shoc_cld_spa_p3_rrtmgp_np1_vs_np4_bfb
184/188 Test #188: homme_shoc_cld_spa_p3_rrtmgp_np1_vs_np4_bfb ...   Passed    0.15 sec
185/188 Test #120: homme_standalone_np1 ..........................   Passed  628.56 sec
        Start 124: homme_np1_vs_np2_bfb
186/188 Test #124: homme_np1_vs_np2_bfb ..........................   Passed    0.07 sec
        Start 125: homme_np1_vs_np3_bfb
187/188 Test #125: homme_np1_vs_np3_bfb ..........................   Passed    0.07 sec
        Start 126: homme_np1_vs_np4_bfb
188/188 Test #126: homme_np1_vs_np4_bfb ..........................   Passed    0.07 sec

99% tests passed, 2 tests failed out of 188

Label Time Summary:
cld             = 6044.97 sec*proc (28 tests)
cld_fraction    =   1.11 sec*proc (1 test)
driver          = 7181.44 sec*proc (23 tests)
dynamics        = 7208.69 sec*proc (29 tests)
fail            = 677.31 sec*proc (20 tests)
io              =  68.22 sec*proc (12 tests)
p3              = 7851.94 sec*proc (70 tests)
physics         = 8933.54 sec*proc (95 tests)
rrtmgp          = 6723.44 sec*proc (38 tests)
shoc            = 6109.91 sec*proc (35 tests)
spa             = 1489.98 sec*proc (26 tests)

Total Test time (real) = 629.04 sec

The following tests FAILED:
	101 - shoc_run_and_cmp_cxx_omp16 (Failed)
	102 - shoc_run_and_cmp_f90_omp16 (Failed)
CMake Error at /home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream/cmake/ctest_script.cmake:79 (message):
  Test had fails


===============================================================================
Testing '61cfb6d20177827afb3f46b9a3aaddda19f071e9' for test 'full_sp_debug'
===============================================================================
RUN: taskset -c 24-47 sh -c 'SCREAM_BUILD_PARALLEL_LEVEL=24 CTEST_PARALLEL_LEVEL=24 ctest -V --output-on-failure --resource-spec-file /home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream/ctest-build/full_sp_debug/ctest_resource_file.json -DNO_SUBMIT=True -DBUILD_WORK_DIR=/home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream/ctest-build/full_sp_debug -DBUILD_NAME_MOD=full_sp_debug -S /home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream/cmake/ctest_script.cmake -DCMAKE_COMMAND="-C /home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream/cmake/machine-files/blake.cmake -DNetCDF_Fortran_PATHS=/home/projects/e3sm/scream/libs/netcdf-fortran/install/blake -DNetCDF_C_PATHS=/ascldap/users/projects/e3sm/scream/libs/netcdf-c/install/blake -DCMAKE_BUILD_TYPE=Debug -DSCREAM_DOUBLE_PRECISION=False -DEKAT_DEFAULT_BFB=True -DEKAT_DISABLE_TPL_WARNINGS='ON' -DSCREAM_AUTOTESTER='ON' -DCMAKE_CXX_COMPILER=/home/projects/x86-64-skylake/openmpi/2.1.2/intel/19.3.199/bin/mpicxx -DCMAKE_C_COMPILER=/home/projects/x86-64-skylake/openmpi/2.1.2/intel/19.3.199/bin/mpicc -DCMAKE_Fortran_COMPILER=/home/projects/x86-64-skylake/openmpi/2.1.2/intel/19.3.199/bin/mpifort -DSCREAM_DYNAMICS_DYCORE=HOMME -DSCREAM_TEST_MAX_TOTAL_THREADS=24 -DSCREAM_TEST_DATA_DIR=/home/projects/e3sm/scream/pr-autotester/master-baselines/blake/full_sp_debug/data" '
FROM: /home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream/ctest-build/full_sp_debug
===============================================================================
Testing '61cfb6d20177827afb3f46b9a3aaddda19f071e9' for test 'release'
===============================================================================
RUN: taskset -c 72-95 sh -c 'SCREAM_BUILD_PARALLEL_LEVEL=24 CTEST_PARALLEL_LEVEL=24 ctest -V --output-on-failure --resource-spec-file /home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream/ctest-build/release/ctest_resource_file.json -DNO_SUBMIT=True -DBUILD_WORK_DIR=/home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream/ctest-build/release -DBUILD_NAME_MOD=release -S /home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream/cmake/ctest_script.cmake -DCMAKE_COMMAND="-C /home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream/cmake/machine-files/blake.cmake -DNetCDF_Fortran_PATHS=/home/projects/e3sm/scream/libs/netcdf-fortran/install/blake -DNetCDF_C_PATHS=/ascldap/users/projects/e3sm/scream/libs/netcdf-c/install/blake -DCMAKE_BUILD_TYPE=Release -DEKAT_DISABLE_TPL_WARNINGS='ON' -DSCREAM_AUTOTESTER='ON' -DCMAKE_CXX_COMPILER=/home/projects/x86-64-skylake/openmpi/2.1.2/intel/19.3.199/bin/mpicxx -DCMAKE_C_COMPILER=/home/projects/x86-64-skylake/openmpi/2.1.2/intel/19.3.199/bin/mpicc -DCMAKE_Fortran_COMPILER=/home/projects/x86-64-skylake/openmpi/2.1.2/intel/19.3.199/bin/mpifort -DSCREAM_DYNAMICS_DYCORE=HOMME -DSCREAM_TEST_MAX_TOTAL_THREADS=24 -DSCREAM_TEST_DATA_DIR=/home/projects/e3sm/scream/pr-autotester/master-baselines/blake/release/data" '
FROM: /home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream/ctest-build/release
===============================================================================
Testing '61cfb6d20177827afb3f46b9a3aaddda19f071e9' for test 'full_debug'
===============================================================================
RUN: taskset -c 0-23 sh -c 'SCREAM_BUILD_PARALLEL_LEVEL=24 CTEST_PARALLEL_LEVEL=24 ctest -V --output-on-failure --resource-spec-file /home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream/ctest-build/full_debug/ctest_resource_file.json -DNO_SUBMIT=True -DBUILD_WORK_DIR=/home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream/ctest-build/full_debug -DBUILD_NAME_MOD=full_debug -S /home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream/cmake/ctest_script.cmake -DCMAKE_COMMAND="-C /home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream/cmake/machine-files/blake.cmake -DNetCDF_Fortran_PATHS=/home/projects/e3sm/scream/libs/netcdf-fortran/install/blake -DNetCDF_C_PATHS=/ascldap/users/projects/e3sm/scream/libs/netcdf-c/install/blake -DCMAKE_BUILD_TYPE=Debug -DEKAT_DEFAULT_BFB=True -DEKAT_DISABLE_TPL_WARNINGS='ON' -DSCREAM_AUTOTESTER='ON' -DCMAKE_CXX_COMPILER=/home/projects/x86-64-skylake/openmpi/2.1.2/intel/19.3.199/bin/mpicxx -DCMAKE_C_COMPILER=/home/projects/x86-64-skylake/openmpi/2.1.2/intel/19.3.199/bin/mpicc -DCMAKE_Fortran_COMPILER=/home/projects/x86-64-skylake/openmpi/2.1.2/intel/19.3.199/bin/mpifort -DSCREAM_DYNAMICS_DYCORE=HOMME -DSCREAM_TEST_MAX_TOTAL_THREADS=24 -DSCREAM_TEST_DATA_DIR=/home/projects/e3sm/scream/pr-autotester/master-baselines/blake/full_debug/data" '
FROM: /home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream/ctest-build/full_debug
===============================================================================
Testing '61cfb6d20177827afb3f46b9a3aaddda19f071e9' for test 'debug_nopack_fpe'
===============================================================================
RUN: taskset -c 48-71 sh -c 'SCREAM_BUILD_PARALLEL_LEVEL=24 CTEST_PARALLEL_LEVEL=24 ctest -V --output-on-failure --resource-spec-file /home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream/ctest-build/debug_nopack_fpe/ctest_resource_file.json -DNO_SUBMIT=True -DBUILD_WORK_DIR=/home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream/ctest-build/debug_nopack_fpe -DBUILD_NAME_MOD=debug_nopack_fpe -S /home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream/cmake/ctest_script.cmake -DCMAKE_COMMAND="-C /home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream/cmake/machine-files/blake.cmake -DNetCDF_Fortran_PATHS=/home/projects/e3sm/scream/libs/netcdf-fortran/install/blake -DNetCDF_C_PATHS=/ascldap/users/projects/e3sm/scream/libs/netcdf-c/install/blake -DCMAKE_BUILD_TYPE=Debug -DSCREAM_PACK_SIZE=1 -DSCREAM_SMALL_PACK_SIZE=1 -DEKAT_DEFAULT_BFB=True -DEKAT_DISABLE_TPL_WARNINGS='ON' -DSCREAM_AUTOTESTER='ON' -DCMAKE_CXX_COMPILER=/home/projects/x86-64-skylake/openmpi/2.1.2/intel/19.3.199/bin/mpicxx -DCMAKE_C_COMPILER=/home/projects/x86-64-skylake/openmpi/2.1.2/intel/19.3.199/bin/mpicc -DCMAKE_Fortran_COMPILER=/home/projects/x86-64-skylake/openmpi/2.1.2/intel/19.3.199/bin/mpifort -DSCREAM_DYNAMICS_DYCORE=HOMME -DSCREAM_TEST_MAX_TOTAL_THREADS=24 -DSCREAM_TEST_DATA_DIR=/home/projects/e3sm/scream/pr-autotester/master-baselines/blake/debug_nopack_fpe/data" '
FROM: /home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream/ctest-build/debug_nopack_fpe
Build type full_debug failed. Here's a list of failed tests:
101:shoc_run_and_cmp_cxx_omp16
102:shoc_run_and_cmp_f90_omp16

Build type full_sp_debug failed. Here's a list of failed tests:
91:shoc_run_and_cmp_cxx_omp16
92:shoc_run_and_cmp_f90_omp16

Build type debug_nopack_fpe failed. Here's a list of failed tests:
101:shoc_run_and_cmp_cxx_omp16
102:shoc_run_and_cmp_f90_omp16

Error(s) occurred during test phase
OVERALL STATUS: FAIL
srun: error: blake24: task 0: Exited with exit code 1
salloc: Relinquishing job allocation 1018997
salloc: Job allocation 1018997 has been revoked.
Starting analysis on blake with cmd: cd /home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream && module purge && module load python/3.8.8/gcc/10.2.0 && module unload gcc/10.2.0 && module load openmpi/2.1.2 zlib git/2.9.4 cmake/3.19.3 && export PATH=/ascldap/users/projects/e3sm/scream/libs/netcdf-fortran/install/blake/bin:$PATH && export PATH=/ascldap/users/projects/e3sm/scream/libs/netcdf-c/install/blake/bin:$PATH && export PATH=/ascldap/users/projects/e3sm/scream/libs/pnetcdf/install/blake/bin:$PATH && export INTEL_LICENSE_FILE=/home/projects/x86-64/intel/licenses/USE_SERVER.lic && export OMP_PROC_BIND=spread && true && salloc -N 1 srun -n1 --preserve-env ./scripts/test-all-scream --baseline-dir AUTO $compiler -c EKAT_DISABLE_TPL_WARNINGS=ON -c SCREAM_AUTOTESTER=ON -p -i -m blake 
RUN: cd /home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream && module purge && module load python/3.8.8/gcc/10.2.0 && module unload gcc/10.2.0 && module load openmpi/2.1.2 zlib git/2.9.4 cmake/3.19.3 && export PATH=/ascldap/users/projects/e3sm/scream/libs/netcdf-fortran/install/blake/bin:$PATH && export PATH=/ascldap/users/projects/e3sm/scream/libs/netcdf-c/install/blake/bin:$PATH && export PATH=/ascldap/users/projects/e3sm/scream/libs/pnetcdf/install/blake/bin:$PATH && export INTEL_LICENSE_FILE=/home/projects/x86-64/intel/licenses/USE_SERVER.lic && export OMP_PROC_BIND=spread && true && salloc -N 1 srun -n1 --preserve-env ./scripts/test-all-scream --baseline-dir AUTO $compiler -c EKAT_DISABLE_TPL_WARNINGS=ON -c SCREAM_AUTOTESTER=ON -p -i -m blake 
FROM: /home/e3sm-jenkins/blake/workspace/SCREAM_PullRequest_Autotester_Blake/2718/scream/components/scream
blake failed
+ [[ 1 != 0 ]]
+ fails=0+1
+ [[ -n '' ]]
+ [[ -n '' ]]
+ [[ 0 == 1 ]]
+ [[ 0 == 1 ]]
+ [[ 1 > 0 ]]
+ exit 1
Build step 'Execute shell' marked build as failure
Performing Post build task...
Match found for : : True
Logical operation result is TRUE
Running script  : #!/bin/bash -le

cd $WORKSPACE/${BUILD_ID}/

./scream/components/scream/scripts/jenkins/jenkins_cleanup.sh
[SCREAM_PullRequest_Autotester_Blake] $ /bin/bash -le /tmp/jenkins6363148799396538083.sh
POST BUILD TASK : SUCCESS
END OF POST BUILD TASK : 0
Sending e-mails to: lbertag@sandia.gov
Finished: FAILURE

</code>
</pre>
</details>

<details>
<summary>
  <b>Console Output (last 100 lines) : SCREAM_PullRequest_Autotester_Weaver # 2623</b> (click to expand)
</summary>
<pre>
<code>
      Start 35: cld_fraction_standalone
35/46 Test #35: cld_fraction_standalone ............   Passed    0.86 sec
      Start 36: spa_stand_alone_np1
36/46 Test #36: spa_stand_alone_np1 ................   Passed   47.84 sec
      Start 37: rrtmgp_standalone_unit
37/46 Test #37: rrtmgp_standalone_unit .............   Passed    1.12 sec
      Start 38: rrtmgp_standalone_np1
38/46 Test #38: rrtmgp_standalone_np1 ..............   Passed    1.47 sec
      Start 39: shoc_cld_p3_rrtmgp_np1
39/46 Test #39: shoc_cld_p3_rrtmgp_np1 .............   Passed   31.72 sec
      Start 40: shoc_cld_spa_p3_rrtmgp_np1
40/46 Test #40: shoc_cld_spa_p3_rrtmgp_np1 .........   Passed   57.49 sec
      Start 41: homme_shoc_cld_p3_rrtmgp_np1
41/46 Test #41: homme_shoc_cld_p3_rrtmgp_np1 .......   Passed   42.36 sec
      Start 42: model_baseline
42/46 Test #42: model_baseline .....................   Passed   38.01 sec
      Start 43: model_initial
43/46 Test #43: model_initial ......................   Passed    7.72 sec
      Start 44: model_restart
44/46 Test #44: model_restart ......................   Passed    8.96 sec
      Start 45: model_restart_check
45/46 Test #45: model_restart_check ................   Passed    0.11 sec
      Start 46: homme_shoc_cld_spa_p3_rrtmgp_np1
46/46 Test #46: homme_shoc_cld_spa_p3_rrtmgp_np1 ...   Passed   65.69 sec

96% tests passed, 2 tests failed out of 46

Label Time Summary:
cld             = 197.26 sec*proc (4 tests)
cld_fraction    =   0.86 sec*proc (1 test)
driver          = 137.86 sec*proc (5 tests)
dynamics        = 139.52 sec*proc (5 tests)
fail            = 428.84 sec*proc (4 tests)
io              =   4.95 sec*proc (3 tests)
p3              = 1016.09 sec*proc (10 tests)
physics         = 1074.19 sec*proc (17 tests)
rrtmgp          = 205.45 sec*proc (8 tests)
shoc            = 198.47 sec*proc (5 tests)
spa             = 108.92 sec*proc (5 tests)

Total Test time (real) = 1358.26 sec

The following tests FAILED:
	 23 - shoc_run_and_cmp_cxx (Failed)
	 24 - shoc_run_and_cmp_f90 (Failed)
CMake Error at /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/2623/scream/components/scream/cmake/ctest_script.cmake:79 (message):
  Test had fails


===============================================================================
Testing '05f7b7a7cfdb2f84ddc392e13eb59c91881fe1dc' for test 'release'
===============================================================================
RUN: taskset -c 104-155 sh -c 'SCREAM_BUILD_PARALLEL_LEVEL=52 CTEST_PARALLEL_LEVEL=1 ctest -V --output-on-failure --resource-spec-file /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/2623/scream/components/scream/ctest-build/release/ctest_resource_file.json -DNO_SUBMIT=True -DBUILD_WORK_DIR=/home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/2623/scream/components/scream/ctest-build/release -DBUILD_NAME_MOD=release -S /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/2623/scream/components/scream/cmake/ctest_script.cmake -DCMAKE_COMMAND="-C /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/2623/scream/components/scream/cmake/machine-files/weaver.cmake -DNetCDF_Fortran_PATHS=/home/projects/e3sm/scream/libs/netcdf-fortran/install/weaver/gcc/8.5.0/cuda/10.1.105 -DNetCDF_C_PATHS=/ascldap/users/projects/e3sm/scream/libs/netcdf-c/install/weaver/gcc/8.5.0/cuda/10.1.105 -DCMAKE_BUILD_TYPE=Release -DEKAT_DISABLE_TPL_WARNINGS='ON' -DSCREAM_AUTOTESTER='ON' -DCMAKE_CXX_COMPILER=/ascldap/users/projects/e3sm/scream/libs/openmpi/install/weaver/gcc/8.5.0/cuda/10.1.105/bin/mpicxx -DCMAKE_C_COMPILER=/ascldap/users/projects/e3sm/scream/libs/openmpi/install/weaver/gcc/8.5.0/cuda/10.1.105/bin/mpicc -DCMAKE_Fortran_COMPILER=/ascldap/users/projects/e3sm/scream/libs/openmpi/install/weaver/gcc/8.5.0/cuda/10.1.105/bin/mpifort -DSCREAM_DYNAMICS_DYCORE=HOMME -DSCREAM_TEST_MAX_TOTAL_THREADS=1 -DSCREAM_TEST_DATA_DIR=/home/projects/e3sm/scream/pr-autotester/master-baselines/weaver/release/data" '
FROM: /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/2623/scream/components/scream/ctest-build/release
===============================================================================
Testing '05f7b7a7cfdb2f84ddc392e13eb59c91881fe1dc' for test 'full_sp_debug'
===============================================================================
RUN: taskset -c 52-103 sh -c 'SCREAM_BUILD_PARALLEL_LEVEL=52 CTEST_PARALLEL_LEVEL=1 ctest -V --output-on-failure --resource-spec-file /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/2623/scream/components/scream/ctest-build/full_sp_debug/ctest_resource_file.json -DNO_SUBMIT=True -DBUILD_WORK_DIR=/home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/2623/scream/components/scream/ctest-build/full_sp_debug -DBUILD_NAME_MOD=full_sp_debug -S /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/2623/scream/components/scream/cmake/ctest_script.cmake -DCMAKE_COMMAND="-C /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/2623/scream/components/scream/cmake/machine-files/weaver.cmake -DNetCDF_Fortran_PATHS=/home/projects/e3sm/scream/libs/netcdf-fortran/install/weaver/gcc/8.5.0/cuda/10.1.105 -DNetCDF_C_PATHS=/ascldap/users/projects/e3sm/scream/libs/netcdf-c/install/weaver/gcc/8.5.0/cuda/10.1.105 -DCMAKE_BUILD_TYPE=Debug -DSCREAM_DOUBLE_PRECISION=False -DEKAT_DEFAULT_BFB=True -DEKAT_DISABLE_TPL_WARNINGS='ON' -DSCREAM_AUTOTESTER='ON' -DCMAKE_CXX_COMPILER=/ascldap/users/projects/e3sm/scream/libs/openmpi/install/weaver/gcc/8.5.0/cuda/10.1.105/bin/mpicxx -DCMAKE_C_COMPILER=/ascldap/users/projects/e3sm/scream/libs/openmpi/install/weaver/gcc/8.5.0/cuda/10.1.105/bin/mpicc -DCMAKE_Fortran_COMPILER=/ascldap/users/projects/e3sm/scream/libs/openmpi/install/weaver/gcc/8.5.0/cuda/10.1.105/bin/mpifort -DSCREAM_DYNAMICS_DYCORE=HOMME -DSCREAM_TEST_MAX_TOTAL_THREADS=1 -DSCREAM_TEST_DATA_DIR=/home/projects/e3sm/scream/pr-autotester/master-baselines/weaver/full_sp_debug/data" '
FROM: /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/2623/scream/components/scream/ctest-build/full_sp_debug
===============================================================================
Testing '05f7b7a7cfdb2f84ddc392e13eb59c91881fe1dc' for test 'full_debug'
===============================================================================
RUN: taskset -c 0-51 sh -c 'SCREAM_BUILD_PARALLEL_LEVEL=52 CTEST_PARALLEL_LEVEL=1 ctest -V --output-on-failure --resource-spec-file /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/2623/scream/components/scream/ctest-build/full_debug/ctest_resource_file.json -DNO_SUBMIT=True -DBUILD_WORK_DIR=/home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/2623/scream/components/scream/ctest-build/full_debug -DBUILD_NAME_MOD=full_debug -S /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/2623/scream/components/scream/cmake/ctest_script.cmake -DCMAKE_COMMAND="-C /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/2623/scream/components/scream/cmake/machine-files/weaver.cmake -DNetCDF_Fortran_PATHS=/home/projects/e3sm/scream/libs/netcdf-fortran/install/weaver/gcc/8.5.0/cuda/10.1.105 -DNetCDF_C_PATHS=/ascldap/users/projects/e3sm/scream/libs/netcdf-c/install/weaver/gcc/8.5.0/cuda/10.1.105 -DCMAKE_BUILD_TYPE=Debug -DEKAT_DEFAULT_BFB=True -DEKAT_DISABLE_TPL_WARNINGS='ON' -DSCREAM_AUTOTESTER='ON' -DCMAKE_CXX_COMPILER=/ascldap/users/projects/e3sm/scream/libs/openmpi/install/weaver/gcc/8.5.0/cuda/10.1.105/bin/mpicxx -DCMAKE_C_COMPILER=/ascldap/users/projects/e3sm/scream/libs/openmpi/install/weaver/gcc/8.5.0/cuda/10.1.105/bin/mpicc -DCMAKE_Fortran_COMPILER=/ascldap/users/projects/e3sm/scream/libs/openmpi/install/weaver/gcc/8.5.0/cuda/10.1.105/bin/mpifort -DSCREAM_DYNAMICS_DYCORE=HOMME -DSCREAM_TEST_MAX_TOTAL_THREADS=1 -DSCREAM_TEST_DATA_DIR=/home/projects/e3sm/scream/pr-autotester/master-baselines/weaver/full_debug/data" '
FROM: /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/2623/scream/components/scream/ctest-build/full_debug
Build type full_debug failed. Here's a list of failed tests:
23:shoc_run_and_cmp_cxx
24:shoc_run_and_cmp_f90

Build type full_sp_debug failed. Here's a list of failed tests:
19:shoc_run_and_cmp_cxx
20:shoc_run_and_cmp_f90

Error(s) occurred during test phase
OVERALL STATUS: FAIL
Starting analysis on weaver with cmd: cd /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/2623/scream/components/scream && module purge && module load cuda/10.1.105 ucx/1.6.0 git/2.10.1 python/3.7.3 cmake/3.18.0 && export LD_LIBRARY_PATH=/ascldap/users/projects/e3sm/scream/libs/mpfr/install/weaver/lib:${LD_LIBRARY_PATH} && export LD_LIBRARY_PATH=/ascldap/users/projects/e3sm/scream/libs/gcc/install/weaver/gcc/8.5.0/lib64:${LD_LIBRARY_PATH} && export LD_LIBRARY_PATH=/ascldap/users/projects/e3sm/scream/libs/gcc/install/weaver/gcc/8.5.0/lib:${LD_LIBRARY_PATH} && export PATH=/ascldap/users/projects/e3sm/scream/libs/gcc/install/weaver/gcc/8.5.0/bin:${PATH} && export PATH=/ascldap/users/projects/e3sm/scream/libs/gcc/install/weaver/gcc/8.5.0/libexec/gcc/powerpc64le-unknown-linux-gnu/8.5.0/:${PATH} && export PATH=/ascldap/users/projects/e3sm/scream/libs/openmpi/install/weaver/gcc/8.5.0/cuda/10.1.105/bin:${PATH} && export PATH=/ascldap/users/projects/e3sm/scream/libs/pnetcdf/install/weaver/gcc/8.5.0/cuda/10.1.105/bin:${PATH} && export PATH=/ascldap/users/projects/e3sm/scream/libs/netcdf-c/install/weaver/gcc/8.5.0/cuda/10.1.105/bin:${PATH} && export PATH=/ascldap/users/projects/e3sm/scream/libs/netcdf-fortran/install/weaver/gcc/8.5.0/cuda/10.1.105/bin:${PATH} && true && bsub -I -q rhel7W -n 4 ./scripts/test-all-scream --baseline-dir AUTO $compiler -c EKAT_DISABLE_TPL_WARNINGS=ON -c SCREAM_AUTOTESTER=ON -p -i -m weaver 
RUN: cd /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/2623/scream/components/scream && module purge && module load cuda/10.1.105 ucx/1.6.0 git/2.10.1 python/3.7.3 cmake/3.18.0 && export LD_LIBRARY_PATH=/ascldap/users/projects/e3sm/scream/libs/mpfr/install/weaver/lib:${LD_LIBRARY_PATH} && export LD_LIBRARY_PATH=/ascldap/users/projects/e3sm/scream/libs/gcc/install/weaver/gcc/8.5.0/lib64:${LD_LIBRARY_PATH} && export LD_LIBRARY_PATH=/ascldap/users/projects/e3sm/scream/libs/gcc/install/weaver/gcc/8.5.0/lib:${LD_LIBRARY_PATH} && export PATH=/ascldap/users/projects/e3sm/scream/libs/gcc/install/weaver/gcc/8.5.0/bin:${PATH} && export PATH=/ascldap/users/projects/e3sm/scream/libs/gcc/install/weaver/gcc/8.5.0/libexec/gcc/powerpc64le-unknown-linux-gnu/8.5.0/:${PATH} && export PATH=/ascldap/users/projects/e3sm/scream/libs/openmpi/install/weaver/gcc/8.5.0/cuda/10.1.105/bin:${PATH} && export PATH=/ascldap/users/projects/e3sm/scream/libs/pnetcdf/install/weaver/gcc/8.5.0/cuda/10.1.105/bin:${PATH} && export PATH=/ascldap/users/projects/e3sm/scream/libs/netcdf-c/install/weaver/gcc/8.5.0/cuda/10.1.105/bin:${PATH} && export PATH=/ascldap/users/projects/e3sm/scream/libs/netcdf-fortran/install/weaver/gcc/8.5.0/cuda/10.1.105/bin:${PATH} && true && bsub -I -q rhel7W -n 4 ./scripts/test-all-scream --baseline-dir AUTO $compiler -c EKAT_DISABLE_TPL_WARNINGS=ON -c SCREAM_AUTOTESTER=ON -p -i -m weaver 
FROM: /home/e3sm-jenkins/weaver/workspace/SCREAM_PullRequest_Autotester_Weaver/2623/scream/components/scream
weaver failed
+ [[ 1 != 0 ]]
+ fails=0+1
+ [[ -n '' ]]
+ [[ -n '' ]]
+ [[ 0 == 1 ]]
+ [[ 0 == 1 ]]
+ [[ 1 > 0 ]]
+ exit 1
Build step 'Execute shell' marked build as failure
Performing Post build task...
Match found for : : True
Logical operation result is TRUE
Running script  : #!/bin/bash -le

cd $WORKSPACE/${BUILD_ID}/

./scream/components/scream/scripts/jenkins/jenkins_cleanup.sh
[SCREAM_PullRequest_Autotester_Weaver] $ /bin/bash -le /tmp/jenkins7750147320187922763.sh
POST BUILD TASK : SUCCESS
END OF POST BUILD TASK : 0
Sending e-mails to: lbertag@sandia.gov
Finished: FAILURE

</code>
</pre>
</details>

<details>
<summary>
  <b>Console Output (last 100 lines) : SCREAM_PullRequest_Autotester_Mappy # 2039</b> (click to expand)
</summary>
<pre>
<code>
180/188 Test #188: homme_shoc_cld_spa_p3_rrtmgp_np1_vs_np4_bfb ...   Passed    0.06 sec
181/188 Test #171: homme_shoc_cld_p3_rrtmgp_np1 ..................   Passed  291.59 sec
        Start 175: homme_shoc_cld_p3_rrtmgp_np1_vs_np2_bfb
182/188 Test #175: homme_shoc_cld_p3_rrtmgp_np1_vs_np2_bfb .......   Passed    0.04 sec
        Start 176: homme_shoc_cld_p3_rrtmgp_np1_vs_np3_bfb
183/188 Test #176: homme_shoc_cld_p3_rrtmgp_np1_vs_np3_bfb .......   Passed    0.04 sec
        Start 177: homme_shoc_cld_p3_rrtmgp_np1_vs_np4_bfb
184/188 Test #177: homme_shoc_cld_p3_rrtmgp_np1_vs_np4_bfb .......   Passed    0.04 sec
185/188 Test #120: homme_standalone_np1 ..........................   Passed  325.12 sec
        Start 124: homme_np1_vs_np2_bfb
186/188 Test #124: homme_np1_vs_np2_bfb ..........................   Passed    0.03 sec
        Start 125: homme_np1_vs_np3_bfb
187/188 Test #125: homme_np1_vs_np3_bfb ..........................   Passed    0.03 sec
        Start 126: homme_np1_vs_np4_bfb
188/188 Test #126: homme_np1_vs_np4_bfb ..........................   Passed    0.03 sec

99% tests passed, 2 tests failed out of 188

Label Time Summary:
cld             = 3723.67 sec*proc (28 tests)
cld_fraction    =   0.53 sec*proc (1 test)
driver          = 4087.59 sec*proc (23 tests)
dynamics        = 4101.15 sec*proc (29 tests)
fail            = 574.22 sec*proc (20 tests)
io              =  61.18 sec*proc (12 tests)
p3              = 5244.14 sec*proc (70 tests)
physics         = 5924.47 sec*proc (95 tests)
rrtmgp          = 4161.45 sec*proc (38 tests)
shoc            = 3780.06 sec*proc (35 tests)
spa             = 890.64 sec*proc (26 tests)

Total Test time (real) = 325.25 sec

The following tests FAILED:
	101 - shoc_run_and_cmp_cxx_omp16 (Failed)
	102 - shoc_run_and_cmp_f90_omp16 (Failed)
CMake Error at /home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream/cmake/ctest_script.cmake:79 (message):
  Test had fails


===============================================================================
Testing 'cfac51028dd65492527aeeb7727c7cc8ec0b4ccf' for test 'full_sp_debug'
===============================================================================
RUN: taskset -c 32-63 sh -c 'SCREAM_BUILD_PARALLEL_LEVEL=32 CTEST_PARALLEL_LEVEL=32 ctest -V --output-on-failure --resource-spec-file /home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream/ctest-build/full_sp_debug/ctest_resource_file.json -DNO_SUBMIT=True -DBUILD_WORK_DIR=/home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream/ctest-build/full_sp_debug -DBUILD_NAME_MOD=full_sp_debug -S /home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream/cmake/ctest_script.cmake -DCMAKE_COMMAND="-C /home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream/cmake/machine-files/mappy.cmake -DNetCDF_Fortran_PATHS=/projects/sems/install/rhel7-x86_64/sems/tpl/netcdf/4.7.3/gcc/9.2.0/openmpi/4.0.2/parallel -DNetCDF_C_PATHS=/projects/sems/install/rhel7-x86_64/sems/tpl/netcdf/4.7.3/gcc/9.2.0/openmpi/4.0.2/parallel -DCMAKE_BUILD_TYPE=Debug -DSCREAM_DOUBLE_PRECISION=False -DEKAT_DEFAULT_BFB=True -DEKAT_DISABLE_TPL_WARNINGS='ON' -DSCREAM_AUTOTESTER='ON' -DCMAKE_CXX_COMPILER=/projects/sems/install/rhel7-x86_64/sems/compiler/gcc/9.2.0/openmpi/4.0.2/bin/mpicxx -DCMAKE_C_COMPILER=/projects/sems/install/rhel7-x86_64/sems/compiler/gcc/9.2.0/openmpi/4.0.2/bin/mpicc -DCMAKE_Fortran_COMPILER=/projects/sems/install/rhel7-x86_64/sems/compiler/gcc/9.2.0/openmpi/4.0.2/bin/mpifort -DSCREAM_DYNAMICS_DYCORE=HOMME -DSCREAM_TEST_MAX_TOTAL_THREADS=32 -DSCREAM_TEST_DATA_DIR=/sems-data-store/ACME/baselines/scream/master-baselines/full_sp_debug/data" '
FROM: /home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream/ctest-build/full_sp_debug
===============================================================================
Testing 'cfac51028dd65492527aeeb7727c7cc8ec0b4ccf' for test 'debug_nopack_fpe'
===============================================================================
RUN: taskset -c 64-95 sh -c 'SCREAM_BUILD_PARALLEL_LEVEL=32 CTEST_PARALLEL_LEVEL=32 ctest -V --output-on-failure --resource-spec-file /home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream/ctest-build/debug_nopack_fpe/ctest_resource_file.json -DNO_SUBMIT=True -DBUILD_WORK_DIR=/home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream/ctest-build/debug_nopack_fpe -DBUILD_NAME_MOD=debug_nopack_fpe -S /home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream/cmake/ctest_script.cmake -DCMAKE_COMMAND="-C /home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream/cmake/machine-files/mappy.cmake -DNetCDF_Fortran_PATHS=/projects/sems/install/rhel7-x86_64/sems/tpl/netcdf/4.7.3/gcc/9.2.0/openmpi/4.0.2/parallel -DNetCDF_C_PATHS=/projects/sems/install/rhel7-x86_64/sems/tpl/netcdf/4.7.3/gcc/9.2.0/openmpi/4.0.2/parallel -DCMAKE_BUILD_TYPE=Debug -DSCREAM_PACK_SIZE=1 -DSCREAM_SMALL_PACK_SIZE=1 -DEKAT_DEFAULT_BFB=True -DEKAT_DISABLE_TPL_WARNINGS='ON' -DSCREAM_AUTOTESTER='ON' -DCMAKE_CXX_COMPILER=/projects/sems/install/rhel7-x86_64/sems/compiler/gcc/9.2.0/openmpi/4.0.2/bin/mpicxx -DCMAKE_C_COMPILER=/projects/sems/install/rhel7-x86_64/sems/compiler/gcc/9.2.0/openmpi/4.0.2/bin/mpicc -DCMAKE_Fortran_COMPILER=/projects/sems/install/rhel7-x86_64/sems/compiler/gcc/9.2.0/openmpi/4.0.2/bin/mpifort -DSCREAM_DYNAMICS_DYCORE=HOMME -DSCREAM_TEST_MAX_TOTAL_THREADS=32 -DSCREAM_TEST_DATA_DIR=/sems-data-store/ACME/baselines/scream/master-baselines/debug_nopack_fpe/data" '
FROM: /home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream/ctest-build/debug_nopack_fpe
===============================================================================
Testing 'cfac51028dd65492527aeeb7727c7cc8ec0b4ccf' for test 'release'
===============================================================================
RUN: taskset -c 96-127 sh -c 'SCREAM_BUILD_PARALLEL_LEVEL=32 CTEST_PARALLEL_LEVEL=32 ctest -V --output-on-failure --resource-spec-file /home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream/ctest-build/release/ctest_resource_file.json -DNO_SUBMIT=True -DBUILD_WORK_DIR=/home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream/ctest-build/release -DBUILD_NAME_MOD=release -S /home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream/cmake/ctest_script.cmake -DCMAKE_COMMAND="-C /home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream/cmake/machine-files/mappy.cmake -DNetCDF_Fortran_PATHS=/projects/sems/install/rhel7-x86_64/sems/tpl/netcdf/4.7.3/gcc/9.2.0/openmpi/4.0.2/parallel -DNetCDF_C_PATHS=/projects/sems/install/rhel7-x86_64/sems/tpl/netcdf/4.7.3/gcc/9.2.0/openmpi/4.0.2/parallel -DCMAKE_BUILD_TYPE=Release -DEKAT_DISABLE_TPL_WARNINGS='ON' -DSCREAM_AUTOTESTER='ON' -DCMAKE_CXX_COMPILER=/projects/sems/install/rhel7-x86_64/sems/compiler/gcc/9.2.0/openmpi/4.0.2/bin/mpicxx -DCMAKE_C_COMPILER=/projects/sems/install/rhel7-x86_64/sems/compiler/gcc/9.2.0/openmpi/4.0.2/bin/mpicc -DCMAKE_Fortran_COMPILER=/projects/sems/install/rhel7-x86_64/sems/compiler/gcc/9.2.0/openmpi/4.0.2/bin/mpifort -DSCREAM_DYNAMICS_DYCORE=HOMME -DSCREAM_TEST_MAX_TOTAL_THREADS=32 -DSCREAM_TEST_DATA_DIR=/sems-data-store/ACME/baselines/scream/master-baselines/release/data" '
FROM: /home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream/ctest-build/release
===============================================================================
Testing 'cfac51028dd65492527aeeb7727c7cc8ec0b4ccf' for test 'full_debug'
===============================================================================
RUN: taskset -c 0-31 sh -c 'SCREAM_BUILD_PARALLEL_LEVEL=32 CTEST_PARALLEL_LEVEL=32 ctest -V --output-on-failure --resource-spec-file /home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream/ctest-build/full_debug/ctest_resource_file.json -DNO_SUBMIT=True -DBUILD_WORK_DIR=/home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream/ctest-build/full_debug -DBUILD_NAME_MOD=full_debug -S /home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream/cmake/ctest_script.cmake -DCMAKE_COMMAND="-C /home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream/cmake/machine-files/mappy.cmake -DNetCDF_Fortran_PATHS=/projects/sems/install/rhel7-x86_64/sems/tpl/netcdf/4.7.3/gcc/9.2.0/openmpi/4.0.2/parallel -DNetCDF_C_PATHS=/projects/sems/install/rhel7-x86_64/sems/tpl/netcdf/4.7.3/gcc/9.2.0/openmpi/4.0.2/parallel -DCMAKE_BUILD_TYPE=Debug -DEKAT_DEFAULT_BFB=True -DEKAT_DISABLE_TPL_WARNINGS='ON' -DSCREAM_AUTOTESTER='ON' -DCMAKE_CXX_COMPILER=/projects/sems/install/rhel7-x86_64/sems/compiler/gcc/9.2.0/openmpi/4.0.2/bin/mpicxx -DCMAKE_C_COMPILER=/projects/sems/install/rhel7-x86_64/sems/compiler/gcc/9.2.0/openmpi/4.0.2/bin/mpicc -DCMAKE_Fortran_COMPILER=/projects/sems/install/rhel7-x86_64/sems/compiler/gcc/9.2.0/openmpi/4.0.2/bin/mpifort -DSCREAM_DYNAMICS_DYCORE=HOMME -DSCREAM_TEST_MAX_TOTAL_THREADS=32 -DSCREAM_TEST_DATA_DIR=/sems-data-store/ACME/baselines/scream/master-baselines/full_debug/data" '
FROM: /home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream/ctest-build/full_debug
Build type full_debug failed. Here's a list of failed tests:
101:shoc_run_and_cmp_cxx_omp16
102:shoc_run_and_cmp_f90_omp16

Build type full_sp_debug failed. Here's a list of failed tests:
91:shoc_run_and_cmp_cxx_omp16
92:shoc_run_and_cmp_f90_omp16

Build type debug_nopack_fpe failed. Here's a list of failed tests:
101:shoc_run_and_cmp_cxx_omp16
102:shoc_run_and_cmp_f90_omp16

Error(s) occurred during test phase
OVERALL STATUS: FAIL
Starting analysis on mappy with cmd: cd /home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream && module purge && module load sems-env sems-gcc/9.2.0 sems-cmake/3.19.1 sems-git/2.10.1 sems-openmpi/4.0.2 sems-netcdf/4.7.3/parallel && export OMP_PROC_BIND=spread && true &&  ./scripts/test-all-scream --baseline-dir AUTO $compiler -c EKAT_DISABLE_TPL_WARNINGS=ON -c SCREAM_AUTOTESTER=ON -p -i -m mappy 
RUN: cd /home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream && module purge && module load sems-env sems-gcc/9.2.0 sems-cmake/3.19.1 sems-git/2.10.1 sems-openmpi/4.0.2 sems-netcdf/4.7.3/parallel && export OMP_PROC_BIND=spread && true &&  ./scripts/test-all-scream --baseline-dir AUTO $compiler -c EKAT_DISABLE_TPL_WARNINGS=ON -c SCREAM_AUTOTESTER=ON -p -i -m mappy 
FROM: /home/e3sm-jenkins/jenkins-ws/workspace/SCREAM_PullRequest_Autotester_Mappy/2039/scream/components/scream
mappy failed
+ [[ 1 != 0 ]]
+ fails=0+1
+ [[ -n '' ]]
+ [[ -n '' ]]
+ [[ 0 == 1 ]]
+ [[ 0 == 1 ]]
+ [[ 1 > 0 ]]
+ exit 1
Build step 'Execute shell' marked build as failure
Performing Post build task...
Match found for : : True
Logical operation result is TRUE
Running script  : #!/bin/bash -le

cd $WORKSPACE/${BUILD_ID}/

./scream/components/scream/scripts/jenkins/jenkins_cleanup.sh
[SCREAM_PullRequest_Autotester_Mappy] $ /bin/bash -le /tmp/jenkins2466056118076873261.sh
POST BUILD TASK : SUCCESS
END OF POST BUILD TASK : 0
Sending e-mails to: lbertag@sandia.gov
Finished: FAILURE

</code>
</pre>
</details>

--- a/components/eam/src/physics/cam/shoc.F90
+++ b/components/eam/src/physics/cam/shoc.F90
@@ -3801,7 +3801,8 @@ subroutine update_host_dse(&
 
   do k=1,nlev
     do i=1,shcol
-      temp = (thlm(i,k)+(lcond/cp)*shoc_ql(i,k))/inv_exner(i,k)
+!      temp = (thlm(i,k)+(lcond/cp)*shoc_ql(i,k))/inv_exner(i,k)
+      temp = (thlm(i,k)/inv_exner(i,k))+(lcond/cp)*shoc_ql(i,k)
       host_dse(i,k) = cp*temp+ggr*zt_grid(i,k)+phis(i)
     enddo
   enddo

--- a/components/eam/src/physics/cam/shoc.F90
+++ b/components/eam/src/physics/cam/shoc.F90
@@ -3801,7 +3801,6 @@ subroutine update_host_dse(&
 
   do k=1,nlev
     do i=1,shcol
-!      temp = (thlm(i,k)+(lcond/cp)*shoc_ql(i,k))/inv_exner(i,k)
       temp = (thlm(i,k)/inv_exner(i,k))+(lcond/cp)*shoc_ql(i,k)
       host_dse(i,k) = cp*temp+ggr*zt_grid(i,k)+phis(i)
     enddo

--- a/components/eam/src/physics/cam/shoc_intr.F90
+++ b/components/eam/src/physics/cam/shoc_intr.F90
@@ -609,7 +609,7 @@ end function shoc_implements_cnst
    real(r8) :: minqn, rrho(pcols,pver), rrho_i(pcols,pverp)    ! minimum total cloud liquid + ice threshold    [kg/kg]
    real(r8) :: cldthresh, frac_limit
    real(r8) :: ic_limit, dum1
-   real(r8) :: inv_exner_surf
+   real(r8) :: inv_exner_surf, pot_temp
  
    real(r8) :: wpthlp_sfc(pcols), wprtp_sfc(pcols), upwp_sfc(pcols), vpwp_sfc(pcols)
    real(r8) :: wtracer_sfc(pcols,edsclr_dim)
@@ -777,7 +777,9 @@ end function shoc_implements_cnst
        um(i,k) = state1%u(i,k)
        vm(i,k) = state1%v(i,k)
        
-       thlm(i,k) = state1%t(i,k)*inv_exner(i,k)-(latvap/cpair)*state1%q(i,k,ixcldliq)
+       !thlm(i,k) = state1%t(i,k)*inv_exner(i,k)-(latvap/cpair)*state1%q(i,k,ixcldliq)
+       pot_temp = state1%t(i,k)*inv_exner(i,k)
+       thlm(i,k) = pot_temp-(pot_temp/state1%t(i,k))*(latvap/cpair)*state1%q(i,k,ixcldliq)
        thv(i,k) = state1%t(i,k)*inv_exner(i,k)*(1.0_r8+zvir*state1%q(i,k,ixq)-state1%q(i,k,ixcldliq)) 
  
        tke_zt(i,k) = max(tke_tol,state1%q(i,k,ixtke))

--- a/components/eam/src/physics/cam/shoc_intr.F90
+++ b/components/eam/src/physics/cam/shoc_intr.F90
@@ -777,7 +777,6 @@ end function shoc_implements_cnst
        um(i,k) = state1%u(i,k)
        vm(i,k) = state1%v(i,k)
        
-       !thlm(i,k) = state1%t(i,k)*inv_exner(i,k)-(latvap/cpair)*state1%q(i,k,ixcldliq)
        pot_temp = state1%t(i,k)*inv_exner(i,k)
        thlm(i,k) = pot_temp-(pot_temp/state1%t(i,k))*(latvap/cpair)*state1%q(i,k,ixcldliq)
        thv(i,k) = state1%t(i,k)*inv_exner(i,k)*(1.0_r8+zvir*state1%q(i,k,ixq)-state1%q(i,k,ixcldliq)) 

